general/sponsors.rst

```diff
@@ -0,0 +1,63 @@
+Sponsors
+==================
```

Comment:
I believe that this page should stay on https://ubports.com/page/sponsors, unless we have a really good reason to have it in our documentation. Remember that people can download this documentation as a PDF or epub file and it is then formatted as a book.

Reply:
+1 The docs should not *replace* but complement ubports.com and ubuntu-touch.io. What's supposed to be replaced is wiki.ubports.com.

Reply:
I agree on this one.

Reply:
We love our sponsors though!  I just wanted to show some extra love.